uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -92,7 +94,7 @@ async def process_request(self, path, headers):
         """
         path_portion, _, query_string = path.partition("?")
 
-        websockets.handshake.check_request(headers)
+        websockets.legacy.handshake.check_request(headers)
```

Comment:
This should fail on `websockets==0.8`, right?

Reply:
yes it does fail, I'm wondering if we should just pin websockets higher ?

Reply:
Either that, or doing something like:
```python
if websockets.__version__ < "0.9":
    handshake = websockets.handshake
else:
   handshake = websockets.legacy.handshake
```
Maybe also adding a warning? Anyway, I'm fine with both.

Reply:
ok now I have given it more thoughts and considering that:
1. we do a minor bump soon
2. a cve has been issued on 8.* and is solved in 9.1
we should just keep this PR the way it currently is and up wesockets min reqs to 9.1

thoughts ?

Reply:
I didn't know about `2.`. So yeah, let's do this: `we should just keep this PR the way it currently is and up wesockets min reqs to 9.1`.